apps/ejabberd/src/ejabberd_c2s.erl

```diff
@@ -1295,6 +1295,8 @@ handle_routed(<<"message">>, From, To, Packet, StateData) ->
         allow ->
             {allow, Packet#xmlel.attrs, StateData};
         deny ->
+            {deny, Packet#xmlel.attrs, StateData};
+        block ->
```

Comment:
There are still some lines in `ejabberd_c2s` module where the branch `block` is not covered when `privacy_check_packet/5` is called eg. here [1] [2] [3]. I was just wondering if we're sure that we don't need to cover them too

[1] https://github.com/esl/MongooseIM/blob/blocking_commands/apps/ejabberd/src/ejabberd_c2s.erl#L2128
[2] https://github.com/esl/MongooseIM/blob/blocking_commands/apps/ejabberd/src/ejabberd_c2s.erl#L1203
[3] https://github.com/esl/MongooseIM/blob/blocking_commands/apps/ejabberd/src/ejabberd_c2s.erl#L1315